dev/io.openliberty.checkpoint_fat/publish/servers/jndiServer/bootstrap.properties

```diff
@@ -0,0 +1,13 @@
+###############################################################################
+# Copyright (c) 2022 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+bootstrap.include=../testports.properties
+com.ibm.ws.logging.trace.specification=*=info:checkpoint=all
+io.openliberty.checkpoint.allowed.features=timedexit-1.0
```

Comment:
I have a PR #22286 that adds the following to all our test servers, can you add this to the bootstrap.properties?

```
io.openliberty.checkpoint.dump.threads=true
```